### ISSUE TITLE
Documenter defArbo.json

### ISSUE THREAD

**Issue by MajorLee95** [poltergeist42/arboProject#4]
Ce serait une bonne idées de documenter ce document.

**Comment by poltergeist42** [comment]
Oui. Si seulement les docstring étaient supporter en JSON (Je sent la
flemme qui se pointe)

**Comment by poltergeist42** [comment]
Ah en faite ça va mieux, la doc est déjà écrite. Il faut juste que je la
mette à jour.

https://poltergeist42.github.io/arboProject/defArbo.html
